### DISCUSSION TITLE
Microsoft Delete Basic Auth (IMAP)

### DISCUSSION THREAD

**Discussion by aleixbiosca** [uvdesk/community-skeleton#609] [Q&A] [answered]
Good morning, If Microsoft removes basic authentication with IMAP, how would this configuration work with UVDESK?
[https://techcommunity.microsoft.com/t5/exchange-team-blog/basic-authentication-deprecation-in-exchange-online-september/ba-p/3609437](https://techcommunity.microsoft.com/t5/exchange-team-blog/basic-authentication-deprecation-in-exchange-online-september/ba-p/3609437)

**Comment by abaum26** [comment]
Microsoft 365 has been requiring authentication via OAuth since the beginning of October. This must be set up in Microsoft Azure and various parameters (Application Client ID, Application Client Secret and for Azure the Application Callback URL) must be entered in the Uvdesk email configuration. I believe this would need to be built into the email configuration interface so that this can be configured appropriately when setting up SMTP and IMAP.

Even if you apply to Microsoft for an extension of the old basic authentication, it will only be supported until the end of the year. Actually, a solution is needed now, since the period for applying for the extension may have expired.

In this respect, I would be very grateful to Uvdesk if they could take this into account in their planning.

Many greetings!

**Comment by komal-sh-27** [comment, marked as answer]
@abaum26 

You should be able to **integrate with your Outlook modern auth** option and also **mailboxes using the new Microsoft Apps section** available on the dashboard which will walk you through the steps to **integrate with Microsoft platforms**. 

This feature will be added with our **next release - V1.2.0** update which will be available by the end of this week. 

Thank you for your time and patience!!!

**Reply by abaum26** [reply]
Thank you for the information.

**Reply by aleixbiosca** [reply]
Hello, do we know what day the update of version 1.2.0 will come out?

**Reply by komal-sh-27** [reply]
> Hello, do we know what day the update of version 1.2.0 will come out?

@aleixbiosca 

We are working on this update and also on working bug fixes to integrate with your Outlook modern auth option. 
We will try to version - 1.2.0 available as soon as possible time.

Thank you for your time and patience!!!

**Reply by aleixbiosca** [reply]
Good morning, Sorry for being insistent and annoying, but having the service with Microsoft, we ran out of service from day 1 of january and we have all the incidents that we manage through your ticketing systems. Do you know when the official version of 1.2 will be uploaded to be able to use MicrosoftApps?

**Reply by aleixbiosca** [reply]
I'm sorry, I have no response from the last email

**Reply by komal-sh-27** [reply]
We are extremely sorry that it's taking so long for the next stable releases to be launched. We're trying to push forward as fast as we can, but there are a lot more changes than we had anticipated originally. We need to release v1.1.2 before we can release v1.2.0, as v1.1.2 itself introduces a lot of internal changes to workflows which is the core feature of our project.

The v1.2 release will also most likely be renamed to v2.0 as it involves a number of breaking changes which won't be compatible with the previous versions in v1.x development line. 


Thank you for your time and patience!!!

**Reply by aleixbiosca** [reply]
Do you have a more or less launch forecast?

**Reply by frntline-dev** [reply]
Is there any update after a year?

**Reply by SyedMuhammadMusa** [reply]
Hello Everyone  @komal-sh-27 
I have been waiting for the Microsoft App feature since a year. Now on the github the latest version is 1.1.6 and I have tested this version and it still does not include this feature can anyone tell me when it will be released?

**Reply by papnoisanjeev** [reply]
@SyedMuhammadMusa New version (v1.1.7) has been released with Microsoft App you can check.
